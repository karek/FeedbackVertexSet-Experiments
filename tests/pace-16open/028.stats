70,85,5
