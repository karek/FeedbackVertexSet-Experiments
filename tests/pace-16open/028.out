8
1 58 39 50 7 68 52 25 
